apps/viewer/src/components/viewer-video-view/video-control-bar/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { Box, Center, HStack, Slider, SliderFilledTrack, SliderThumb, SliderTrack } from '@chakra-ui/react';
+import React from 'react';
+
+import {
```

Comment:
There are lots of duplicate codes and logic in publisher-video-view and viewer-video-view. We'd better move these into the video-view component for future maintenance.